sdks/python/build.gradle

```diff
@@ -77,11 +77,14 @@ task preCommitPy2() {
   dependsOn "lint"
 }
 
-task portablePreCommit() {
+addPortableWordCountTask('BatchPy2', false)
```

Comment:
We can infer batch or streaming from the flag

Reply:
We can, however, the portableWordCountExample definition would be more complicated. Do we meed that task at all? how is it used? 
I can remove it if you prefer and remove one of the prefix parameter.

Reply:
We can certainly remove portableWordCountExample but we would like to keep portableWordCount

Reply:
Done.